env.source.sample

```diff
@@ -25,6 +25,10 @@ export OCM_USER=${OCM_USER:-your_user}
 ### Your kerberos username (without domain) if different than your OCM_USER
 export OCM_CONTAINER_KERBEROS_USER=${OCM_CONTAINER_KERBEROS_USER:-${OCM_USER:-$(whoami)}}
 
+### Set the default namespace for velero to avoid using 
+### --namespace=openshift-velero each time
+velero client config set namespace=openshift-velero
```

Comment:
I like the change, but can we put this in the supplemental bashrc in `container-setup/utils/bashrc_supplement.sh`?  

I worry that adding it here in the .env.source _might_ run into issues when attempting to build or run the container if you don't have velero installed on your host.

Reply:
Thanks, fixed now :)